### DISCUSSION TITLE
bulb returning to white automatically

### DISCUSSION THREAD

**Discussion by LinkinStg** [arendst/Tasmota#12341] [Q&A] [answered]
hello, I have a former tuya bulb now a tasmota one, it wotks perfect from the tasmota ip gui, already seted up in home assistant, but I can only power on and of from home assistant, whenever I try to change it on the color wheel the bulb returns to white automatically, any Idea where could it be the problem? thanks in advance

**Comment by ascillato** [comment, marked as answer]
Please, open a new issue and share the logs. Please, in the issue complete all the fields of the troubleshooting template in order to have all the information. Thanks.

**Reply by sfromis** [reply]
Reading the logs, you should have good chances of finding out what happens, ans solve it yourself.

**Reply by LinkinStg** [reply]
Already fixed, for anyone with this issue, I turned of the auto discovery feature in tasmota setoption19 0, an edited the config,yalm in home assistant whit a template, also changed the rgbww values with decimal ones with setoption17 1, and worked like a charm

**Comment by blakadder** [comment]
Or simply let the Tasmota integration autodiscover the light and set it up in HA accordingly
